README.adoc

```diff
@@ -50,15 +30,6 @@ You can take a look at the link:sonar-project.properties[sonar-project.propertie
 - https://docs.sonarcloud.io/advanced-setup/languages/c-c-objective-c/#analysis-steps-using-compilation-database[Using compilation database (compile_commands.json)]
 - https://docs.sonarcloud.io/enriching/test-coverage/test-coverage-parameters/[Test coverage parameters]
 
-= Windows\MSBuild
```

Comment:
Why remove this section?

Reply:
as agreed, closing this

Reply:
After f2f discussion. It is no longer relevant.